### DISCUSSION TITLE
How to get the relationship filed as placeholder instead of select

### DISCUSSION THREAD

**Discussion by khanakia** [filamentphp/filament#8822] [Help] [answered]
When i use the relationship as select then it works but when I want to use it as Placeholder on the right column it does not work

So how do i get the relationship field as a placeholder instead of showing it as select?

#### Working
```php
Forms\Components\Select::make('order_status_id')
      ->relationship('order_status', 'name')
      ->required()
```

####  Not working just empty
```php
Forms\Components\Placeholder::make('order_status.name')->label('Order Status'),
```

![Screenshot 2023-10-02 at 3 08 59 PM](https://github.com/filamentphp/filament/assets/2174170/8d69ef54-96cb-476f-9111-e921dc73b7be)

**Comment by leandrocfe** [comment]
You could use an [infolist](https://filamentphp.com/docs/3.x/panels/resources/viewing-records#using-an-infolist-instead-of-a-disabled-form)

**Reply by khanakia** [reply]
But filament does not allow you to use the infolist inside the form function I have mixed data some fields will be editable and some are read-only.

```php
 public static function form(Form $form): Form {
}
 ```

**Reply by leandrocfe** [reply, marked as answer]
```php
Forms\Components\Placeholder::make('order_status')
    ->content(fn ($record): string => $record->order_status->name)
    ->label('Order Status')
```

**Reply by khanakia** [reply]
Thanks
